src/ScrumBoardItBundle/DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('complexity_field')
                             ->isRequired()
                         ->end()
+                        ->scalarNode('printed_tag')
```

Comment:
Is this necessary? Is this a vendor that need exposed configuration? http://symfony.com/doc/current/best_practices/configuration.html#semantic-configuration-don-t-do-it